src/test/java/com/zufar/onlinestore/product/util/ProductStub.java

```diff
@@ -27,7 +28,27 @@ public static ProductListWithPaginationInfoDto buildSampleProducts(Integer page,
         return productList;
     }
 
-    private static List<ProductInfoDto> generateSampleProducts() {
+    public static ProductInfoDto generateSampleProduct() {
+        ProductInfoDto productA = new ProductInfoDto();
+        productA.setId(UUID.randomUUID());
+        productA.setName("Product A");
+        productA.setDescription("Description for Product A");
+        productA.setPrice(BigDecimal.valueOf(10.50));
+        productA.setQuantity(20);
+        return productA;
+    }
+
+    public static ProductInfo generateSampleEntityProduct() {
+        ProductInfo productA = new ProductInfo();
+        productA.setProductId(UUID.randomUUID());
+        productA.setName("Product A");
+        productA.setDescription("Description for Product A");
+        productA.setPrice(BigDecimal.valueOf(10.50));
+        productA.setQuantity(20);
+        return productA;
+    }
+
+    public static List<ProductInfoDto> generateSampleProducts() {
```

Comment:
We can use library for generate test data. 
For example:
https://github.com/DiUS/java-faker